### ISSUE TITLE
static ip example?

### ISSUE THREAD

**Issue by eriksape** [msolters/wifi-control-node#19]
Hi!

There is an example to config with static ip?

**Comment by msolters** [comment]
No, this library really only requests WiFi interface-specific actions.  It does not assume or attempt to manipulate IP assignment.  IMO that varies too greatly by OS.  Even within Linux, say, there are so many different ways to configure an interface to be static or DHCP.  But really, the SSID association has nothing to do with IP addresses.  That's a different level of the network stack entirely.
